app/test_split_integer.py

```diff
@@ -1,21 +1,25 @@
 from app.split_integer import split_integer
 
 
-def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
-
-
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    goals = split_integer(6, 2)
+
+    assert goals == [3, 3]
```

Comment:
I thin no need to create variable `goals`
```suggestion
    assert split_integer(6, 2) == [3, 3]
```